apph-front/src/utils/translations/fr/common.json

```
@@ -0,0 +1,194 @@
+      "nameOverChar": "Le nom ou le prénom ne peuvent pas dépasser les 127 caractères.",
+      "emailOverChar": "L'email ne peut pas dépasser les 255 caractères.",
+      "loginOverChar": "Le login ne peut pas dépasser les 255 caractères."
+    },
+    "create": "Créer un compte",
+    "exist": "Vous avez déjà un compte ? Se connecter",
+    "created": "Utilisateur crée"
+  },
+  "user": {
+    "email": "Email",
+    "login": "Identifiant",
+    "firstName": "Prénom",
+    "lastName": "Nom",
+    "password": "Mot de passe",
+    "passwordConfirmation": "Confirmation du mot de passe",
+    "error": {
+      "passwordNotMatch": "Les mots de passe de correspondent pas",
+      "notAuthenticated": "L'utilisateur n'est pas authentifié",
+      "sessionBindUserNotExist": "L'utilisateur lié à cette session n'existe pas",
+      "expiredSession": "La session a expiré. Veuillez vous reconnecter",
+      "loginAlreadyToken": "This login is already taken",
+      "notExist": "User does not exist"
+    }
+  },
+  "photoTable": {
+    "title": "Titre",
+    "description": "Description",
+    "creationDate": "Date de création",
+    "shootingDate": "Date de prise de vue",
+    "lastModification": "Date de dernière modification",
+    "size": "Taille",
+    "tags": "Tags",
+    "url": "Url",
+    "actions": "Actions",
+    "error": "Argument illégal."
+  },
+  "field": {
+    "photos": "Mes Photos",
+    "folders": "Mes Dossiers",
+    "trips": "Mes Voyages",
+    "tags": "Mes Tags",
+    "treatments": "Mes traitements",
+    "advancedSearch": "Recherche avancée"
+  },
+  "folder": {
+    "create": "Créer",
+    "delete": "Supprimer du dossier",
+    "createFolder": "Créer un dossier",
+    "creation": "Création d'un dossier",
+    "enterName": "Entrez le nom de dossier:",
+    "name": "Nom du Dossier",
+    "new": "Nouveau Dossier",
+    "moveInto": "Déplacer dans le dossier",
+    "moveTo": "Déplacer vers un dossier",
+    "move": "Déplacer",
+    "confirmDeletePhoto": "Si vous confirmez, la photo sera supprimée du dossier",
+    "successMove": "Le déplacement des photos est terminé.",
+    "error": {
+      "error": "Error",
+      "oneOf": {
+        "photoNotExist": "L'une des photos n'existe pas.",
+        "notBelongUser": "L'une des photos n'appartient pas à l'utilisateur.",
+        "alreadyExist": "L'une des photos est déjà dans le dossier.",
+        "existingName": "L'une des photos comporte un nom existant déjà dans le dossier destinataire."
+      },
+      "nullFolder": "Folder est null !",
+      "emptyFolder": "Le nom du dossier ne peux pas être vide",
+      "existingFolder": "Le dossier existe déjà dans le dossier actuel.",
+      "notFound": "Dossier parent introuvable.",
+      "unauthorized": "L'utilisateur n'a pas accès à ce dossier.",
+      "root": "Impossible de créer un dossier racine.",
+      "userNotExist": "L'utilisateur n'existe pas.",
+      "parentNotExist": "Dossier parent non existant.",
+      "warning": "Avertissement",
```

ça dépend des utilisation, mais peut être ça plutôt
```suggestion
      "warning": "Attention",
```